components/base/SubmitButton.vue

```diff
@@ -19,22 +20,27 @@
 
 <script setup lang="ts">
 import { NeoButton, NeoField } from '@kodadot1/brick'
+import { NeoButtonVariant } from '@kodadot1/brick'
 
 export interface Props {
   disabled?: boolean
   expanded?: boolean
+  disabledIcon?: boolean
```

Comment:
That wouldn't work because it has a default value (`paper-plane`). Should I remove the default value, but that might break functionality on other pages.